CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 **Changes in version 1.25**
 
 - Fixed wrong messages for rules.
+- Removed unused property.
```

Comment:
This is too vague, developers reading this will have no idea what you mean. Can you please change it to this:

Removed unused input TX status.